<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogrammtransformation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="fpsLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QDial" name="dial">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="notchTarget">
             <double>3.700000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="threadCountLabel">
            <property name="text">
             <string>Threads: 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="LiveImageView" name="imageView"/>
      </item>
      <item>
       <widget class="LiveImageView" name="transformedImageView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="histogramView">
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TransformCurveView" name="transformView">
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Zurücksetzen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gammaTransform">
        <property name="text">
         <string>Gamma-Transformation (2,2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="transformedHistogramView">
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_ffnen">
   <property name="text">
    <string>Öffnen...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TransformCurveView</class>
   <extends>QGraphicsView</extends>
   <header>transformcurveview.h</header>
   <slots>
    <signal>transformChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>LiveImageView</class>
   <extends>QGraphicsView</extends>
   <header>liveimageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
